climpred/smoothing.py

```diff
@@ -0,0 +1,230 @@
+import numpy as np
+
+try:
+    import xesmf as xe
+except ImportError:
+    xe = None
+
+
+def spatial_smoothing_xesmf(
+    ds,
+    d_lon_lat_dict={'lon': 5, 'lat': 5},
+    method='bilinear',
+    periodic=False,
+    filename=None,
+    reuse_weights=True,
+):
+    """
+    Quick regridding
```

Comment:
Can you update this to be more descriptive?

Reply:
I will but maybe we dont even need this long function. see other comment

Reply:
reworked docstrings